src/Cell.java

```diff
@@ -1,2 +1,32 @@
 public class Cell {
+
+    private boolean isAlive;
+    private String cell;
+
+    public Cell (){
+        this.isAlive = false;
+        this.cell = initialiseCell();
+    }
+
+    public boolean getIsAlive() {
+        return isAlive;
+    }
+
+    public String getCell() {
+        return cell;
+    }
+
+    private String initialiseCell(){
+        if(!this.isAlive){
+            cell = "x";
+        }
+        else {
+            cell = "o";
+        }
+        return cell;
```

Comment:
A good question to think about is what these two variables represent. We're keeping them in sync, but they actually do represent two separate things, which I know is why you've got them here.

What is your reasoning for having both `isAlive` and `cell`?

I think if you want you could try just implementing the function to toggle the state between dead and alive (and vice versa) with the cell as it currently is to see if we want these things to be in sync.